src/openscm_runner/adapters/ciceroscm_adapter/make_scenario_files.py

```diff
@@ -0,0 +1,236 @@
+"""
+Module with functionality to make emission input files
+"""
+import csv
+import logging
+import os
+
+import numpy as np
+import pandas as pd
+
+# Method to convert the unit names form tonnes to grams
+
+
+def unit_name_converter(unit):
+    """
+    Convert unit names between tonnes and grammes
+    """
+    uprefix = unit[0:2]
+    if uprefix == "Mt":
+        return "Tg{}".format(unit[2:])
+    if uprefix == "kt":
+        return "Gg{}".format(unit[2:])
+    if uprefix == "Gt":
+        return "Pg{}".format(unit[2:])
+    return unit
+
+
+# Method to find unit conversion factor
+
+
+def unit_conv_factor(cicero_unit, unit, comp):
+    """
+    Find conversion factor between two units
+    """
+    # Todo: raise exception
+    conv_dict = {"P": 1.0e15, "T": 1.0e12, "G": 1.0e9, "M": 1.0e6, "k": 1.0e3}
+    conv_factor = conv_dict[unit[0]] / conv_dict[cicero_unit[0]]
+    if unit[1:] != cicero_unit[1:]:
+        if (
+            unit[1:] == "g CO2/yr"
+            and cicero_unit[1:] == "g_C"
+            and (comp in ("CO2", "CO2_lu"))
+        ):
+            conv_factor = conv_factor * 3.0 / 11  # Carbon mass fraction in CO2
+        elif unit[1:] == "g N2O/yr" and cicero_unit[1:] == "g_N" and comp == "N2O":
+            conv_factor = conv_factor * 0.636  # Nitrogen mass fraction in N2O
+        elif unit[1:] == "t NOx/yr" and cicero_unit[1:] == "t_N" and comp == "NOx":
+            conv_factor = (
+                conv_factor * 0.304
+            )  # Nitrogen mass fraction in NOx (approximated by NO2)
+        elif unit[1:] == "g SO2/yr" and cicero_unit[1:] == "g_S" and comp == "SO2":
+            conv_factor = conv_factor * 0.501  # Sulphur mass fraction in SO2
+
+    return conv_factor
+
+
+def read_ssp245_em(ssp245_em_file):
```

Comment:
```suggestion
def _read_ssp245_em(ssp245_em_file):
```

To make clear it's not part of the public API (might also apply to the functions above)

Reply:
Done